src/gmpints.h

```diff
@@ -70,6 +70,17 @@ extern Obj ObjInt_Int(Int i);
 extern Obj ObjInt_UInt(UInt i);
 extern Obj ObjInt_Int8(Int8 i);
 
+/**************************************************************************
+**
+** The following functions a GAP integer into an Int, UInt, Int8 or UInt8
```

Comment:
Do we really want to duplicate this comment (and then maintain it in two places)? Perhaps just add it here, in the header, and remove the copy in the .c file, resp. change the comment there to just say something to the effect of "For details, see the .h file"